Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -501,10 +501,9 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         completion: @escaping (Result<EvaluationResult, Error>) -> Void
     ) throws {
         let manifestPreamble: ByteString
-        switch toolsVersion {
-        case .vNext:
+        if toolsVersion >= .v5_8 {
```

Comment:
Wouldn't this have to be 5.9 for the `main` branch?  Otherwise the behaviour of 5.8 packages will be different between the 5.8 branch and `main`.

Reply:
I'm proposing that we would enable this for 5.8. If we think that's correct, I'd also cherry-pick to 5.8

Reply:
I see.  In that case, yes then this logic looks correct, but only if we also take it in 5.8.